src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerOtherTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.adjourncase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.ABOUT_TO_SUBMIT;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+
+class AdjournCaseAboutToSubmitHandlerOtherTest extends AdjournCaseAboutToSubmitHandlerTestBase {

```

Comment:
This has been renamed to AdjournCaseAboutToSubmitCanHandleTest  - please rebase

Reply:
fixed